src/Appwrite/Platform/Tasks/CalcTierStats.php

```diff
@@ -356,4 +201,209 @@ public function action(Group $pools, Cache $cache, Database $dbForConsole, Regis
             Console::error("Message could not be sent. Mailer Error: {$mail->ErrorInfo}");
         }
     }
+
+
+    private function getData(Document $project, Database $dbForConsole, Database $dbForProject): array
+    {
+
+            $stats['Project ID'] = $project->getId();
+            $stats['Organization ID']   = $project->getAttribute('teamId', null);
+
+            $teamInternalId = $project->getAttribute('teamInternalId', 0);
+
+            $team = $dbForConsole->findOne('teams', [
+                Query::equal('$internalId', [(string)$teamInternalId]),
+            ]);
+
+            $stats['Organization Email'] = '';
+
+        if ($team !== false && !$team->isEmpty()) {
+            $userInternalId = $team->getAttribute('userInternalId', 0);
+
+            $user = $dbForConsole->findOne('users', [
+                Query::equal('$internalId', [(string)$userInternalId]),
+            ]);
+
+            if ($user !== false && !$user->isEmpty()) {
+                $stats['Organization Email'] = $user['email'];
+            }
+        }
```

Comment:
We need to add an else condition and log it so we know how many ( and which ) teams fall into this category

Also please run the linter to fix the indentation